src/core/Notification/BaseNotification/BaseNotification.baseStyles.ts

```diff
@@ -0,0 +1,73 @@
+import { css } from 'styled-components';
+import { SuomifiTheme } from '../../theme';
+import { element, font } from '../../theme/reset';
+
+export const baseNotificationBaseStyles = (theme: SuomifiTheme) => css`
+  &.fi-notification {
+    ${element(theme)}
+    ${font(theme)('bodyTextSmall')}
+    p {
+      font-size: 16px;
+    }
+    & .fi-notification_label {
+      ${font(theme)('bodySemiBold')}
+      margin-top: -4px;
+      margin-bottom: ${theme.spacing.xxs};
+    }
+
+    padding-bottom: 17px;
+    width: 100%;
+    & .fi-notification_style-wrapper {
+      display: flex;
+      align-items: flex-start;
+    }
+    & .fi-notification_icon-wrapper {
+      margin-top: 17px;
+      & .fi-notification_icon {
+        height: 24px;
+        width: 24px;
+      }
+    }
+    & .fi-notification_text-content-wrapper {
+      display: flex;
+      flex: 3 1 auto;
+      flex-direction: column;
+      padding: 9px 20px 0 20px;
+      margin-top: ${theme.spacing.s};
+      margin-bottom: 0;
+      & .fi-notification_content {
+        vertical-align: middle;
+        ${font(theme)('bodyTextSmall')}
+      }
+    }
+    box-shadow: 0px 4px 8px 0px rgba(41, 41, 41, 0.2);
```

Comment:
The internally used shadows are implemented in shadows.ts and used from there. Check if one of the existing ones would suit the purpose or add a new one in there if needed.

Reply:
At least check if the shadow used in Expander is suitable for this case also; it seems to be close.

What you think about it, would it be suitable?

Reply:
I created now one, because size of shadow is different. Now it is:
notificationBoxShadow: `0px 4px 8px 0px ${alphaHex(0.14)(colors.blackBase)}`,